collie/metrics.py

```diff
@@ -15,83 +15,69 @@
 from collie.model import BasePipeline
 
 
-def _get_user_item_pairs(model: BasePipeline,
-                         user_or_item_ids: Union[np.array, torch.tensor],
-                         n_items_or_users: int,
-                         device: Union[str, torch.device]) -> Tuple[torch.tensor, torch.tensor]:
+def _get_row_col_pairs(row_ids: Union[np.array, torch.tensor],
+                       n_cols: int,
+                       device: Union[str, torch.device]) -> Tuple[torch.tensor, torch.tensor]:
     """
-    Create tensors pairing each input user ID with each item ID or vice versa.
+    Create tensors pairing each input row ID with each column ID.
 
     Parameters
     ----------
-    model: collie.model.BasePipeline
-        Model that can take a (user_id, item_id) pair as input and return a recommendation score
-    user_or_item_ids: np.array or torch.tensor, 1-d
+    row_ids: np.array or torch.tensor, 1-d
         Iterable[int] of users or items to score
-    n_items_or_users: int
+    n_cols: int
```

Comment:
Should we also change the description of this variable on the next line to not say `items or users` and just be consistent with `row/col`?

Reply:
Ditto to a lot of docstrings in this file.

Reply:
Oh yes, apologies I meant to do that before submitting!